BringgSDK.js

```diff
@@ -937,7 +940,7 @@ var BringgSDK = (function () {
   module._setDriverActivity = function (newActivity) {
     driverActivity = google.maps.TravelMode.DRIVING;
 
-    switch (newActivity) {
+    switch (parseInt(newActivity)) {
       case 1: //not moving
```

Comment:
this is not related to your PR per-se but it can affect you.
i think this line is wrong, and should set the driver activity according to the default driver activity rather than always driving.

Reply:
comment above